.gitlab-ci.yml

```diff
@@ -42,7 +42,7 @@ onboarding_nodejs:
       matrix:
         - ONBOARDING_FILTER_ENV: [dev, prod]
           ONBOARDING_FILTER_WEBLOG: [test-app-nodejs]
-          SCENARIO: [HOST_AUTO_INJECTION, HOST_AUTO_INJECTION_INSTALL_SCRIPT]
+          SCENARIO: [HOST_AUTO_INJECTION, HOST_AUTO_INJECTION_INSTALL_SCRIPT, HOST_AUTO_INJECTION_INSTALL_SCRIPT_PROFILING]
```

Comment:
I originally tried it with SIMPLE_HOST_AUTO_INJECTION_PROFILING and its counterpart SIMPLE_HOST_AUTO_INJECTION isn't featured here, only HOST_AUTO_INJECTION is, that's why it isn't added.

I figured we could add it to individual library repos, maybe even replace use of SIMPLE_HOST_AUTO_INJECTION there with SIMPLE_HOST_AUTO_INJECTION_PROFILING in those repos where it's supported if we only want to run a single test in them as it tests both tracing _and_ profiling.

Of course, under your proposed solution for refactoring vm provisions and weblogs that only differ in env vars, it will become unnecessary.

Reply:
Ok, added test-app-nodejs-profiling with SIMPLE_HOST_AUTO_INJECTION_PROFILING